hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBaseRelation.scala

```diff
@@ -571,6 +568,20 @@ abstract class HoodieBaseRelation(val sqlContext: SQLContext,
           val readerSchema = StructType(requiredDataSchema.structTypeSchema.fields ++ partitionSchema.fields)
 
           (parquetReader, readerSchema)
+        case HoodieFileFormat.ORC =>
```

Comment:
@ymZhao1001 : Can you add a testcase to showcase ORC reader working successfully